pnpm-workspace.yaml

```diff
@@ -84,6 +84,8 @@ catalog:
   "@types/node": ^22.14.0
   "@types/react": ^19.1.0
   "@types/react-dom": ^19.1.1
+  "@types/react-window": ^1.8.8
+  "@types/swagger-ui-react": ^5.18.0
```

Comment:
It doesn't look like you ended up using any of the dependencies you added here unless I'm missing something -- should we revert these changes?